apps/kevin-malone/bot/common/threads/onboarding.py

```diff
@@ -73,7 +70,33 @@ async def control_hook(self, message, user_id):
                 user_id, self.cache, USER_DB_ID_CACHE_KEY, user["id"]
             )
             return StepKeys.ASSOCIATE_EXISTING_USER_WITH_GUILD.value
-        return StepKeys.PROMPT_USER_WALLET_ADDRESS.value
+        return StepKeys.PROMPT_USER_CONNECT_WALLET.value
+
+
+class PromptUserToConnectWallet(BaseStep):
+    """Step to submit wallet address for the govrn profile"""
+
+    name = StepKeys.PROMPT_USER_CONNECT_WALLET.value
+
+    def __init__(self, cache, guild_id, bot):
+        super().__init__()
+        self.cache = cache
+        self.guild_id = guild_id
+        self.bot = bot
+
+    async def send(self, message, user_id):
+        channel = message.channel
+        discord_user = await self.bot.fetch_user(user_id)
+        msg = (
+            "It doesn't look like your wallet"
```

Comment:
can we extract this copy in to config.py? There are a few examples in there of how to format so that we can insert the right link.

Reply:
Yes, we can move the copy to the config.py

Reply:
Going to keep this here as it uses the display name variable. It seems a bit premature to abstract now